test/EndToEnd/tests/InstallPackageTest.ps1

```diff
@@ -458,9 +458,6 @@ function Test-AddBindingRedirectToWebsiteWithNonExistingOutputPath {
 }
 
 function Test-InstallCanPipeToFSharpProjects {
-    [SkipTest('https://github.com/NuGet/Home/issues/11358')]
-    param($context)
```

Comment:
Why did you remove param in half the places but kept in others? 

Can you please make it consistent.

Reply:
It's due to previous [skip test PR](https://github.com/NuGet/NuGet.Client/pull/4328/files) made these inconsistent changes, I just reverted it original state. 
<img width="610" alt="image" src="https://user-images.githubusercontent.com/8766776/154131320-34dbf973-046c-4d98-a56f-d6c2d8508c28.png">

Reply:
But you kept it in https://github.com/NuGet/NuGet.Client/pull/4459/files#diff-46b97442b93a948bc0a165779563f604f433c90a49f58b7372e7500b5c867c22. 

I think my argument we either sue param or we don't.
You're changing that code anyways, and since it's line deletion it won't even affect any of the git blame-ability of the code.

Reply:
Made them uniform.